### PR TITLE
Qa and edit hardhat docs

### PR DESCRIPTION
**Pull request by latechwriter**
**What changed? Why?**

* All of the docs within https://docs.base.org/base-camp/docs/hardhat-setup-overview/hardhat-setup-overview-sbs have been QAed.
* All videos were checked.
* Fixed formatting in a few places as well.

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Review by zencephalon — APPROVED** [review]

**Review by mdehoog — DISMISSED** [review]

File: apps/base-docs/base-camp/docs/hardhat-deploy/hardhat-deploy-sbs.md [hunk 1]
```diff
@@ -68,99 +66,94 @@ We need to modify our tsconfig.json file to look like:
...
+```typescript
+const lockDeployment = await deployments.get('Lock');
 ```
 
-Then we reuse `Lock__factory` but we use the connect function and pass the address of the newly created contract plus a signer.
-
-Now we simply run `npx hardhat test`
-
-And we should get the same result:
+Resue `Lock__factory` but use the connect function and pass the address of the newly-created contract plus a signer. Then, run `npx hardhat test` and you should get the same result:
```
Comment (briandoyle81CB) [inline apps/base-docs/base-camp/docs/hardhat-deploy/hardhat-deploy-sbs.md:156]: Should be "reuse"

File: apps/base-docs/base-camp/docs/hardhat-forking/hardhat-forking.md [hunk 2]
```diff
@@ -38,15 +38,15 @@ Hardhat forking also has other capabilities like:
 - hardhat_setCode
 - hardhat_setStorageAt
 
-Those won't be covered in this guide, however we recommend exploring them a bit more in the following link:
+Those won't be covered in this guide, however it's recommend to explor them a bit more in the following link:
```
Comment (briandoyle81CB) [inline apps/base-docs/base-camp/docs/hardhat-forking/hardhat-forking.md:41]: "explore"

File: apps/base-docs/base-camp/docs/hardhat-deploy/hardhat-deploy-sbs.md (hunk 1 above)
Comment (latechwriter) [reply apps/base-docs/base-camp/docs/hardhat-deploy/hardhat-deploy-sbs.md]: Fixed!

File: apps/base-docs/base-camp/docs/hardhat-forking/hardhat-forking.md (hunk 2 above)
Comment (latechwriter) [reply apps/base-docs/base-camp/docs/hardhat-forking/hardhat-forking.md]: Fixed!

**Review by briandoyle81CB — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for briandoyle81CB @ 2023-10-13 21:19:36 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by briandoyle81CB — APPROVED** [review]

**Review by zencephalon — APPROVED** [review]

**Pull request merged**